### PR TITLE
Add README file for my-sites/drafts

### PR DESCRIPTION
**Pull request by lancewillett**

### CODE REVIEW

File: client/my-sites/drafts/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Drafts
+======
+
+The view module for the Drafts post list in `/posts/drafts` as a React component. See also the single `Draft` component.
```
Comment (mtias) [inline client/my-sites/drafts/README.md]: This actually powers the drafts panel in the editor and `/drafts`, not `/posts/drafts`. The latter is via `client/my-sites/posts`.

So this folder has both the section routes for `/draft`, the controller, the main view for the sections, and a list component that we reuse in the editor. It's a bit messy. :)

Comment (lancewillett) [inline client/my-sites/drafts/README.md]: OK — feel free to edit and push. :)

**Comment by lancewillett** [comment]
Updated based on @mtias feedback and squashed commits.

**Comment by lancewillett** [comment]
Oops. Squashing I lost my last edit. Redoing. :)

**Comment by lancewillett** [comment]
Ping @mtias for a review.

**Comment by mtias** [comment]
:ship:

**Pull request merged**